README.md

```diff
@@ -26,12 +26,13 @@ const warning = require('process-warning')()
 #### Methods
 
 ```
-warning.create(name, code, message)
+warning.create(name, code, message, unlimited)
```

Comment:
Can you make it an `object` option to preserve extensibility in future?

Reply:
Done - added types too